docs/releases/v2019.1.rst

```diff
@@ -15,6 +15,7 @@ possible.
 With Gluon v2019.1, nodes will not answer respondd queries on ``[ff02::2:1001]:1001`` anymore. Respondd
 querier setups still using this address must be updated to the new address ``[ff05::2:1001]:1001``
 (supported since Gluon v2017.1). This change was required due to cross-domain leakage of respondd data.
+(hopglass-server needs an update to work with the new address, so make sure you use the latest version.)
```

Comment:
If you are using hopglass to query respondd data you need to update hopglass to at least $VERSION.

Don't use brackets.